postgres/src/postgres.cpp

```diff
@@ -0,0 +1,109 @@
+#include "cocaine/storage/postgres.hpp"
+
+#include <cocaine/dynamic.hpp>
+
+namespace cocaine {
+namespace storage {
+
+postgres_t::postgres_t(context_t& context, const std::string& name, const dynamic_t& args) :
+    api::storage_t(context, name, args),
+    table_name(args.as_object().at("pg_table_name", "cocaine_index").as_string()),
+    wrapped(api::storage(context, args.as_object().at("pg_underlying_storage", "core").as_string())),
+    pg_pool(args.as_object().at("pg_pool_size", 1ul).as_uint(),
+            args.as_object().at("pg_connection_string", "").as_string())
```

Comment:
Without connection string it's useless, even with the empty one.

Maybe better to explicitly check the `pg_connection_string` sanity in the ctor body?